src/android/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ target_sources(${LRS_TARGET}
         "${CMAKE_CURRENT_LIST_DIR}/jni/options.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/jni/advanced_mode.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/jni/utils.cpp"
+        "${CMAKE_CURRENT_LIST_DIR}/jni/jni_usercb.cpp"
```

Comment:
Sorry for not agreeing with this name - please consider frame_callback.cpp instead of jni_usercb.cpp